src/test/java/TopNavigationTest.java

```diff
@@ -0,0 +1,150 @@
+import components.TopNavigation;
+import enums.Attribute;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import utils.WebDriverUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TopNavigationTest {
+    private static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
+    private static final By topNavigationBarCategoryNameFirstLevel = By.cssSelector("ul a > span:not(.ui-menu-icon)");
+    private static final By topNavigationBarCategoryFirstLevelLink = By.cssSelector("ul a");
```

Comment:
Both selectors should be directly used in validation method, as they appear in only one place.